shell.client/src/main/java/org/metaborg/spoofax/shell/client/IEditor.java

```diff
@@ -36,4 +36,20 @@
      */
     List<String> history();
 
+    /**
+     * Set the prompt to display.
+     *
+     * @param promptString
+     *            The prompt string.
+     */
+	void setPrompt(String promptString);
+
+    /**
+     * Set the prompt to display when in multi-line mode.
+     *
+     * @param promptString
+     *            The prompt string.
+     */
+	void setContinuationPrompt(String promptString);
+
```

Comment:
I don't know if we should do this by the way, but we can keep it like this for now. Probably it's fine.